pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -500,11 +506,7 @@ func getPrinter(format string) *editPrinterOptions {
 	}
 }
 
-func (o *EditOptions) visitToPatch(
-	originalInfos []*resource.Info,
-	patchVisitor resource.Visitor,
-	results *editResults,
-) error {
+func (o *EditOptions) visitToPatch(originalInfos []*resource.Info, patchVisitor resource.Visitor, results *editResults) error {
```

Comment:
just style change right ? I remember previously it was only one line, and somebody just change to three lines, and now change it back. do we have a signature style in [coding-conventions.md](https://github.com/kubernetes/community/blob/master/contributors/devel/coding-conventions.md) to illustrate which way is better ?